vms/job/tests.py

```diff
@@ -1,15 +1,14 @@
-from django.contrib.auth.models import User
 from django.test import TestCase
+
 from event.models import Event
 from job.models import Job
 from job.services import *
```

Comment:
Here I left import * because for each function from job.services there must be a test